src/dao/PlatformEmitterDAO.js

```diff
@@ -1,7 +1,27 @@
 import AbstractContractDAO from './AbstractContractDAO'
+import TransactionModel from '../models/TransactionModel'
+import web3utils from 'web3/lib/utils/utils'
+import ContractsManagerDAO from '../dao/ContractsManagerDAO'
+import ERC20DAO from '../dao/ERC20DAO'
 
 export default class PlatformEmitterDAO extends AbstractContractDAO {
   constructor (at) {
     super(require('chronobank-smart-contracts/build/contracts/ChronoBankPlatformEmitter.json'), at)
   }
+
+  static async prepareTxsMap (txs): Promise<Map<TransactionModel>> {
+    const daoPromises = txs.map((tx) => ContractsManagerDAO.getERC20DAOBySymbol(web3utils.toAscii(tx.args.symbol)))
```

Comment:
use this._c.bytesToString instead of web3Utils
you should not use such middleware utils in your business logic code